extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -180,6 +181,18 @@ require(['search-tracking', 'uuid', 'wikia.trackingOptIn'], function(searchTrack
 				$elem.attr('href', modifiedUrl);
 			});
 		},
+		searchScopeEvents: function() {
+			var searchForm = $('#search-v2-form'),
+				scopes = $('.SearchInput .wds-list a');
+
+			// Show and hide video filter options when radio buttons change.
+			scopes.on('click', function(e) {
+				var value = $(e.target).attr('data-value');
+
+				searchForm.find('input[name=scope]').first().val(value);
```

Comment:
if you have `id` for the element, it is always better to get this element by id `#search-v2-scope`. Also there is no need to use `first()`